src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java

```diff
@@ -0,0 +1,56 @@
+package pl.byczazagroda.trackexpensesappbackend.model;
+
+import lombok.*;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.*;
+import java.io.Serial;
+import java.util.Date;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@Builder
+@AllArgsConstructor
+@Table(name = "financial_transactions_categories")
+public class FinancialTransactionCategory {
+    /**
+     * Class version 0.1.0.  SerialVersionUID needs to be updated with any change.
+     */
+    @Serial
+    private static final long serialVersionUID = 100000L;
```

Comment:
You added this field for serialization reasons, so please this class should implement the Serializable interface well.